recipes/rubinenv/meta.yaml

```diff
@@ -0,0 +1,138 @@
+{% set name = "rubinenv" %}
+{% set version = "0.1.0" %}
+{% set build_number = 1 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  skip: True         # [python_impl == 'pypy' or win]
+  number: {{ build_number }}
+
+# Since all requirements are required "at run time" for the
+# rubin developers, all requirements are fundamentally
+# run requirements. However, we've attempted to organize 
+# requirements as build-like, host-like, or run-like, in 
+# order inform any evolution of this recipe.
+requirements:
+  # conda-forge-pinning dependencies should also be in host
+  # https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml
+  host:
```

Comment:
This host env won't appear in any downstream environments made with this package or when this package is used in other builds. What is the purpose of the host env?

Reply:
misunderstanding, fixed now.